### DISCUSSION TITLE
Export sequence PNG Transparent

### DISCUSSION THREAD

**Discussion by mamatism** [keiwando/toonsquid#307] [Ideas]
I want export sequence PNG transparent, is it possible?
I see no option for that, please help if it possible.

thanks

**Comment by Cat-Butler** [comment]
Turn off the background layer (eye icon in the timeline) and export to Image Sequence, PNG, Save to Files - output files should automatically have transparency in any areas not covered by paint. Personally, I prefer saving to Files then importing into an app rather than sharing directly to another app.

Note: preview icons in Files will still show a white background, because the Files app doesn't show transparency in the thumbnail icons, but opening one or more of the PNGs in a more capable app (photo manipulation e.g. Affinity Photo, video editor e.g. LumaFusion) shows the transparency.

**Comment by mamatism** [comment]
thank you it works!
